sdk-python/livescoreapi.py

```diff
@@ -10,11 +10,50 @@
 
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
+    
+    get_all()
+    
+    """
+    def __init__(self,key,secret):
+        self.key = key 
+        self.secret = secret 
+    
+    def _get_html_file(self,link):
+        """returns the source code given a link"""
+    
+        try:
+            html_file = urllib.request.urlopen(link)
+            return html_file
+        except Exception as e:
+            print(str(e))
```

Comment:
It would be nice for here to throw another Exception maybe our own exception so the end user of the SDK can catch it otherwise they will not know that the request failed. This is exception is pretty much critical